Controller/Views/CriteriaController.php

```diff
@@ -674,13 +671,15 @@ public function addCriteriaRevision(View $view, array $rawData, $targetFieldName
             ]);
 
             return $revision;
-        } else if ($result['hits']['total'] == 1) {
+        } else if ($searchResponse->getTotal() == 1) {
             /**@var Revision $revision*/
             $revision = null;
-            if (isset($loadedRevision[$result['hits']['hits'][0]['_id']])) {
-                $revision = $loadedRevision[$result['hits']['hits'][0]['_id']];
+            $firstId = $searchResponse->getDocumentCollection()->first()->getId();
```

Comment:
can't call getId() on nullable (first might return null). However you are kind of checking this. Is phpstan complaining or not locally on this line?

Reply:
Also fixed